### PR TITLE
Thief Breaching Charges Balance

### PR DESCRIPTION
**Pull request by Unkn0wnGh0st333**
## About the PR
Replaces C4 with Breaching Charges

## Why / Balance
I believe this was a needed change due to the lethality that C4 can cause. This was made in attempt to prevent Thief murder due to C4 being a pacifist implant bypass. The Breaching Charge is a structural targeting explosive that has near to none player damage unless on the same tile, even then not even crit'ing a completely unarmored person. 

Due to Breaching Charges being weaker though, a third charge was added to the kit to compensate. Though is it better suited for the kits whole point of existing, to breach into areas and steal items.

I'd love to talk more in detail about this if you guys have differing opinions

Edit: In testing breaching charge does about 90 damage to unarmored crew on top of it, while the C4 does staggeringly much more with insta crit and bleed

## Technical details
Replaces C4 with Breaching Charges, Changes description of Breacher Kit on toolbox

## Media
image
image
Edit: Fixed the description saying 2 to 3^

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Nothing that really effects anything code wise

**Changelog**
🆑 
 - Thief's Breacher Kit's C4 replaced with Breaching Charges

### CODE REVIEW

**Comment by Unkn0wnGh0st333** [comment]
WOOPS MINOR WORDING MISTAKE AAAAA

**Comment by deltanedas** [comment]
bad grammar in the ftl + should try this upstream first + ftl has no deltav comment above it

**Comment by MilonPL** [comment]
> bad grammar in the ftl + should try this upstream first + ftl has no deltav comment above it

wdym upstream
breaching charges are a deltav thing

**Comment by Unkn0wnGh0st333** [comment]
@deltanedas Breaching charges are delta only and uhh shut up! ooo gottem

Edit: I'll fix the grammer but uhh I'm not sure how I would leave a comment on an FTL file?

**Comment by deltanedas** [comment]
you have to put the comment above it, on the same line wont be commented it stays :trollface:

**Comment by Unkn0wnGh0st333** [comment]
hopefully I understood what you meant by that and this doesn't fail horrifically

File: Resources/Locale/en-US/thief/backpack.ftl [hunk 1]
```diff
@@ -23,10 +23,11 @@ thief-backpack-category-chameleon-description =
     Disguise as anyone and anything.
 
 thief-backpack-category-tools-name = breacher kit
+#Delta V: C4 replaced with breaching charges and description changed
```
Comment (deltanedas) [inline Resources/Locale/en-US/thief/backpack.ftl]: using \r\n or something its red on github

Comment (Unkn0wnGh0st333) [reply Resources/Locale/en-US/thief/backpack.ftl]: uhh wdym?

Comment (TadJohnson00) [reply Resources/Locale/en-US/thief/backpack.ftl]: Add a space between "#" and "Delta"

**Review by TadJohnson00 — CHANGES_REQUESTED** [review]

**Review by TadJohnson00 — APPROVED** [review]

**Pull request merged**